fabric-entity-attribute-api-v1/src/main/java/net/fabricmc/fabric/mixin/entity/attribute/DefaultAttributeRegistryAccessor.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.entity.attribute;
+
+import java.util.Map;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.attribute.DefaultAttributeContainer;
+import net.minecraft.entity.attribute.DefaultAttributeRegistry;
+
+@Mixin(DefaultAttributeRegistry.class)
+public interface DefaultAttributeRegistryAccessor {
+	@Accessor("DEFAULT_ATTRIBUTE_REGISTRY")
+	static Map<EntityType<? extends LivingEntity>, DefaultAttributeContainer> getRegistry() {
+		throw new AssertionError("mixin dummy");
```

Comment:
Hmm, do we have a consistent message for this type of dummy error across the whole api? I see we seem to have multiple cases of this across the place.

One idea I had was an instance of an exception that we should throw in the `fabric-base-api` so we only refer to one common message we know.

Reply:
I dont think it really needs its own exception, having them all the same would be nice, prob out of scope for this PR.

Reply:
> One idea I had was an instance of an exception that we should throw in the `fabric-base-api` so we only refer to one common message we know.

Do you mean that we should expose the standard thrown error in the base api? Cannot agree, this is more impl-oriented and isn't really useful

Reply:
No standard error, I'd throw UnsupportedOperationException("dummy method, impl to be swapped by mixin") or similar.